### ISSUE TITLE
Update list of mining pools

### ISSUE THREAD

**Issue by oyvkva** [Anoncoin/anoncoin-website#13]
Several people have asked where to point their miners.

Pools are listed here, but it has not been updated in a while:
https://anoncoin.net/Mining_pools/

If anyone feel like checking the pools here and adding the new ones that'd be awesome. There are some new ones in the Anoncoin thread on Bitcoin talk I believe.

If someone starts a new pool they can make a pull request to add it to the list on the website as well.

**Comment by ilovelinux** [comment]
It seems that iSpace and Prohashing no longer support Anoncoin, we should remove them from the list

**Comment by oyvkva** [comment]
Yep, and there is probably some new pools not listed in the Anoncoin thread.
